samples/convert-upload-images-powervs/convert-upload-images-powervs

```diff
@@ -0,0 +1,742 @@
+#!/usr/bin/env bash
+: '
+Copyright (C) 2021 IBM Corporation
+Licensed under the Apache License, Version 2.0 (the “License”);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an “AS IS” BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'
+#-------------------------------------------------------------------------
+set -e
+#set -x
+
+source <(curl -L https://raw.githubusercontent.com/ocp-power-automation/openshift-install-power/master/openshift-install-powervs 2> /dev/null | sed  's/main "$@"//g')
+
+function help {
+  cat <<-EOF
+
+Automation for creating RHCOS/RHEL images to PowerVS Services. This is a wrapper for pvsadm tool.
+
+Usage:
+  ./convert-upload-images-powervs [ --rhel-url <url> | --rhcos-url <url> ] --service-name  <service name> --region <bucket region> --cos-bucket <bucket name> --cos-resource-group <resource group> [ --cos-instance-name <cos instance name> ]
+
+Args:
+      --service-name string       A service name in cloud.ibm.com you have access(Mandatory)
+      --region string             Object store bucket region(Mandatory)
+      --cos-bucket                Object store bucket name(Mandatory)
+      --cos-resource-group        COS resource group(Mandatory)
+      --cos-instance-name         COS instance name(optional)
+      --rhel-url  url             url pointing to the rhel qcow2 image(optional)
+      --rhcos  url                url pointing to the rhcos qcow2 image(optional)
+      --help                      help for upload
+
+EOF
+  exit 0
+}
+
+ARTIFACTS_VERSION="master"
+
+
+
+TF='./terraform'
+CLI_PATH='./ibmcloud'
+TMP_DIR=$(mktemp -d tmp.XXXXXXXXXXX)
+
+ARTIFACTS_DIR="automation"
+LOGFILE="$(date "+%Y%m%d%H%M%S")"
+GIT_URL="https://github.com/ocp-power-automation/infra"
+
+TRACE=0
+TF_TRACE=0
+FORCE_DESTROY=1
+
+NO_OF_RETRY=${NO_OF_RETRY:-"5"}
+SLEEP_TIME=10
+REBOOT_TIMEOUT=30
+
+PVSADM_VERSION="v0.1.1-alpha.2"
+IMAGE_SIZE="120"
+
+# Default Centos image name
+CENTOS_VM_IMAGE_NAME='Linux-CentOS-8-3'
+
+
+#-------------------------------------------------------------------------
+# Check and run setup
+#-------------------------------------------------------------------------
+function precheck_tools {
+  # Run setup if no artifacts
+  if [[ ! -d $ARTIFACTS_DIR ]]; then
+    warn "Cannot find artifacts directory... running setup command"
+    setup
+  else
+    setup_tools
+  fi
+}
+
+#-------------------------------------------------------------------------
+# Display the cluster access information
+#-------------------------------------------------------------------------
+function cluster_access_info {
```

Comment:
Since we are not creating cluster, we should change the function name to reflect the purpose better

Reply:
`cluster_access_info`  function get called from two functions in https://raw.githubusercontent.com/ocp-power-automation/openshift-install-power/master/openshift-install-powervs . One way to solve this is by adding those functions in this script , which will increase the number of lines. I tried aliasing as well but the call redirects to the actual `cluster_access_info` in the `openshift-install-powervs` , not to the one there in this script.